src/list.coffee

```diff
@@ -56,6 +57,11 @@ class List extends Command
       tree packages, (pack) =>
         packageLine = pack.name
         packageLine += "@#{pack.version}" if pack.version?
+        if pack.apmInstallSource?.type is 'git'
+          repo = getRepository(pack)
+          shaLine = "##{pack.apmInstallSource.sha.substr(0, 8)}"
```

Comment:
GitHub will auto-link the `@` version to the commit, but not the `#` version. I don't know how much that matters, but it's a point in favor of `@` :stuck_out_tongue_closed_eyes: 

BinaryMuse/atom-dev-tools#19c33bd1

BinaryMuse/atom-dev-tools@19c33bd1